graspologic/utils/utils.py

```diff
@@ -701,3 +701,79 @@ def plug_in_variance_estimator(x):
         return covariances
 
     return plug_in_variance_estimator
+
+
+def remove_vertices(graph, indices, return_vertices=False):
```

Comment:
this function wasn't originally in scope, but it was so useful while I was testing/experimenting that I think it's worth adding in. It makes the code in the tutorial a lot more simple as well.

Reply:
im fine with adding but have comments on implementation (see other comments)